requirements/common.txt

```diff
@@ -150,3 +150,11 @@ ujson==1.33
 
 uritemplate==0.6
 zope.interface==4.1.2
+talon==1.2.10
+cchardet==1.0.0
+cssselect==0.9.2
+lxml==3.6.0
+numpy==1.11.0
+regex==2016.6.19
+scikit-learn==0.16.1
+scipy==0.17.1
```

Comment:
I saw this traceback go by while installing these packages in our virtualenv.
```
  Partial import of sklearn during the build process.
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-build-cD7HNg/scikit-learn/setup.py", line 173, in <module>
      setup_package()
    File "/tmp/pip-build-cD7HNg/scikit-learn/setup.py", line 165, in setup_package
      from numpy.distutils.core import setup
  ImportError: No module named numpy.distutils.core
  
  ----------------------------------------
  Failed building wheel for scikit-learn
```
Not sure if that's a real problem where we're missing a dependency we should have, or what.

Also, maybe it's worth putting these in a separate requirements/email-mirror.txt file that we include from `common.txt`?  Just for the purpose of giving clarity about why these are dependencies.